samples/server/petstore/nodejs-express-server/api/openapi.yaml

```diff
@@ -8,7 +8,7 @@ info:
   title: OpenAPI Petstore
   version: 1.0.0
 servers:
-- url: http://petstore.swagger.io/v2
+- url: /api/v2
```

Comment:
This should contain the full base path, right?

Ref: https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/test/resources/3_0/petstore.yaml#L3